packages/dds/tree/src/shared-tree/schematizingTreeView.ts

```diff
@@ -89,6 +89,10 @@ export class SchematizingSimpleTreeView<in out TRootSchema extends ImplicitField
 		public readonly nodeKeyManager: NodeKeyManager,
 		public readonly breaker: Breakable = new Breakable("SchematizingSimpleTreeView"),
 	) {
+		if (checkout.forest.anchors.slots.has(ContextSlot)) {
+			throw new UsageError("Cannot create a second view from the same branch");
```

Comment:
that doesn't seem like it will work for all scenarios. It will only work when Thjis will only work if the simple tree view is "viewable" and has a `CheckoutFlexTreeView`. If its not viewable the anchor slot won't have a conext and creating a new simple view will be allowed and can fail later. Is there a better check we can perform?

Reply:
We would have to store some additional state somewhere to make it work.

For example add a ViewSlot that works like ContextSlot, except stores the simple-tree-view. That should be a pretty easy change, so I think its worth adding in this PR to make this check robust.

Reply:
you can define and use the slot in this file, so it shouldn't even need to be exported

Reply:
Okay, let me do that. I wasn't sure if doing that was a good idea :-).